library_conventions/symbols.adoc

```diff
@@ -49,6 +49,22 @@ text elements). For non-symmetrical symbols it should be as close as
 possible to the center, but still on the 2.54mm grid.
 
 
+[#libraryconventions-symbols-outline]
+==== Outline
+
+The outline of a regular symbol should be drawn with a rectangle or a polygon.
+All vertices should be located on the 2.54mm grid and following properties
+should be used:
+
+- *Layer*: _Outlines_
+- *Line Width*: _0.2 mm_
+- *Filled*: _no_
```

Comment:
Shouldn't the symbol outline be filled?

Reply:
No, the yellow fill comes from the "grab area" property, not from the "fill" property.

Reply:
Ah! :slightly_smiling_face: